src/extract_topic.py

```diff
@@ -1,64 +1,233 @@
-            text_data.append(data)
-    return text_data
-
-
-class LemmaCountVectorizer(CountVectorizer):
-"""a count vectorizer with lemmatizer build in"""
-    def build_analyzer(self):
-        lemm = WordNetLemmatizer()
-        analyzer = super(LemmaCountVectorizer, self).build_analyzer()
-        return lambda doc: (lemm.lemmatize(w) for w in analyzer(doc))
-
-
-def process_text2vec(text):
-    cnt_vectorizer = LemmaCountVectorizer(max_df=0.95,
-                                         min_df=2,
-                                         stop_words='english',
-                                         decode_error='ignore')
-    cnt_mat = cnt_vectorizer.fit_transform(text)
-    return cnt_mat, cnt_vectorizer
-
-
-def model(cnt_mat, n_comp = 4):
-    lda = LatentDirichletAllocation(n_components=n_comp, max_iter=5,
-                                learning_method = 'online',
-                                learning_offset = 50.,
-                                random_state = 0)
-    lda.fit(cnt_mat)
-    return lda
-
-
-def print_top_words(model, feature_names, n_top_words=10):
-    for index, topic in enumerate(model.components_):
-        message = "\nTopic #{}:".format(index)
-        message += " ".join([feature_names[i] for i in topic.argsort()[:-n_top_words - 1 :-1]])
-        print(message)
-        print("="*70)
-
-
-def print_topic(text_dir, out_dir, n_top_words = 10):
-    text = generate_text_li(text_dir)
-    tf, tf_vectorizer = process_text2vec(text)
-    lda = model(tf, n_comp = 4)
-    #print("\n Assigning Topics in LDA model: ")
-    feature_names = tf_vectorizer.get_feature_names()
-    with open(out_dir+'/topic.txt', 'w') as f:
-        for index, topic in enumerate(lda.components_):
-            f.writelines("\n"+" ".join([feature_names[i] for i in topic.argsort()[:-n_top_words - 1 :-1]]))
-            
- 
+import re
+import nltk
+from pytorch_pretrained_bert import BertTokenizer
+import time
+from bert_serving.client import BertClient
+from bert_serving.server import BertServer
+from bert_serving.server.helper import get_args_parser
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+
+def bert_tokens(text_all, token_choice='bert-base-uncased'):
+    tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+    text_article_tokenized = []
+    text_flat_tokenized = []
+    for text in text_all:
+        text_tokenized =[]
+        tmp_tokenized=[]
+        for sentence in text:
+            toks = tokenizer.tokenize(sentence)
+            for tok in toks:
+                tmp_tokenized.append(tok)
+            text_tokenized.append(toks)
+        text_flat_tokenized.append(tmp_tokenized)
+        text_article_tokenized.append(text_tokenized)
+    return text_flat_tokenized, text_article_tokenized
+
+
+def get_topic_embedding(topics, port=6000, port_out=6001, model_path='/home/ubuntu/bert_tests/bert-as-service/uncased_L-12_H-768_A-12'):
+    common = [
```

Comment:
Please add a simple docstr to explain what this does. What is `topic` and what's the return values.